.pa11yci.conf.js

```diff
@@ -1,3 +1,40 @@
+var dashboardSettings = [
```

Comment:
There is any reason why the scenarios like `dashboardSettings` are not shared between `.pa11yci.conf.js` and `.pa11yci-pr.conf.js`? They are the same, aren't they?

Reply:
More or less the same, but the difference is that one of them has the "thresholds" property and the other doesn't (We need thresholds for the PR pipeline) :)